packages/web-components/src/kumo/progress-bar/progress-bar.styles.ts

```diff
@@ -0,0 +1,101 @@
+import { css } from '@microsoft/fast-element';
+import { display, forcedColorsStylesheetBehavior } from '../../utils/index.js';
+import {
+  borderRadiusMedium,
+  colorCompoundBrandBackground,
+  colorNeutralBackground6,
+  colorPaletteDarkOrangeBackground3,
+  colorPaletteGreenBackground3,
+  colorPaletteRedBackground3,
+  colorTransparentBackground,
+} from '../../theme/design-tokens.js';
+import { errorState, largeState, successState, warningState } from '../../styles/states/index.js';
+
+/** ProgressBar styles
+ * @public
+ */
+export const styles = css`
+  ${display('block')}
+
+  :host {
+    width: 100%;
+    height: 2px;
+    overflow-x: hidden;
+    background-color: ${colorNeutralBackground6};
+    border-radius: ${borderRadiusMedium};
+    contain: content;
+  }
+
+  :host(${largeState}) {
```

Comment:
We still using size for kumo?

Reply:
There are two "thicknesses" in Kumo and this would equal `thickness="large"`